packages/chrome/src/containers/AccordionContainer.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import PropTypes from 'prop-types';
+import {
+  ControlledComponent,
+  IdManager,
+  composeEventHandlers
+} from '@zendeskgarden/react-selection';
+
+export default class AccordionContainer extends ControlledComponent {
+  static propTypes = {
+    /**
+     * Whether the accordion is currently expanded
+     */
+    expanded: PropTypes.bool,
+    /**
+     * @param {Object} newState - The updated state
+     * @param {Any} newState.expanded - Whether the accordion is currently expanded
+     */
+    onStateChange: PropTypes.func,
+    /**
+     * @param {Object} renderProps
+     * @param {Function} renderProps.getHeadingProps - Props to be spread onto each heading element.
+     * @param {Function} renderProps.getHeadingButtonProps - Props to be spread onto each heading button element.
+     * @param {Function} renderProps.getPanelProps - Props to be spread onto each panel element.
+     * @param {Any} renderProps.expanded - Whether the accordion is currently expanded
+     */
+    children: PropTypes.func,
+    /**
+     * Identical to children
+     */
+    render: PropTypes.func,
+    /**
+     * The root ID to use for descendants. A unique ID is created if none is provided.
+     **/
+    id: PropTypes.string
+  };
+
+  constructor(...args) {
+    super(...args);
+
+    this.state = {
+      expanded: false,
+      id: IdManager.generateId('garden-accordion-container')
+    };
+  }
+
+  getHeaderId = () => `${this.getControlledState().id}-header`;
+
+  getPanelId = () => `${this.getControlledState().id}-panel`;
+
+  getHeadingProps = ({ role = 'heading', ...other } = {}) => {
+    return {
+      role,
```

Comment:
@austin94 per https://www.w3.org/TR/wai-aria-practices/#accordion, the `role="heading"` also needs a value for `aria-level`. Can this be addressed so that the consumer can feed in that value (and in the demo so that it passes the axe audit)?

Reply:
Voiceover was smart enough to auto-detect the level, didn't realize that AXE was complaining.

This is now a required attribute and prop for the `AccordionContainer` and the `CollapsibleSubNavItem`.